### PR TITLE
Removed support for  JAVA_VERSION variable

### PR DESCRIPTION
**Pull request by LongyuZhang**
- Removed support for JAVA_VERSION
Issue: #563

Signed-off-by: Longyu Zhang <longyu.zhang@ibm.com>

### CODE REVIEW

**Comment by LongyuZhang** [comment]
reviewers @llxia @sophia-guo

**Comment by LongyuZhang** [comment]
Test Links:
sanity.system: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_Sandbox/572/
extended.system: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_Sandbox/573/
sanity.functional.regular: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_Sandbox/574/
extended.functional.regular: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_Sandbox/575/

**Comment by sophia-guo** [comment]
Some left over in functional/MBCS_Tests sub folders. Please also take a look at.

File: functional/MBCS_Tests/codepoint/build.xml [hunk 1]
```diff
@@ -27,23 +27,16 @@ limitations under the License.
...
-           <and>
-             <not>
-               <equals arg1="${JAVA_VERSION}" arg2="SE80" />
-             </not>
-             <not>
-               <equals arg1="${JDK_VERSION}" arg2="8" />
-             </not>
-           </and>
-        </condition>
+	<condition property="not_se80" value="false" else="true">
```
Comment (llxia) [inline functional/MBCS_Tests/codepoint/build.xml]: We should rename this property as we do not have SE80 anymore.

File: functional/MBCS_Tests/CLDR_11/build.xml [hunk 2]
```diff
@@ -31,7 +31,7 @@ limitations under the License.
 		<mkdir dir="${build}" />
 	</target>
 
-	<target name="compile" depends="init" description="Using ${JAVA_VERSION} java compile the source  ">
+	<target name="compile" depends="init" description="Using ${JDK_VERSION} java compile the source  ">
```
Comment (llxia) [inline functional/MBCS_Tests/CLDR_11/build.xml]: Just for the readability, can we change this to `Using java${JDK_VERSION} to compile the source `

File: functional/MBCS_Tests/codepoint/build.xml [hunk 3]
```diff
@@ -27,23 +27,16 @@ limitations under the License.
...
+			<equals arg1="${JDK_VERSION}" arg2="8" />
+	</condition>
```
Comment (sophia-guo) [inline functional/MBCS_Tests/codepoint/build.xml]: Personally I feel this is quite confusing using double negtive. I would prefer 
	
        <condition property="not_se80" value=""true">
		<not>	
                       <equals arg1="${JDK_VERSION}" arg2="8" />
               </not>
	</condition>
Probably could also rename property as '8above'?

**Comment by LongyuZhang** [comment]
"JAVA_VERSION" in functional/MBCS_Tests have also been replaced. "not_se80" has been renamed to "jdk8above", and the logic has been modified as suggested, with description changed to ```Using java${JDK_VERSION} to compile the source```. Thanks.

**Pull request merged**